languages/emf-metamodel-dynamic/src/main/java/de/jplag/emf/dynamic/DynamicMetamodelLanguage.java

```diff
@@ -8,7 +9,7 @@
  * created token set instead of a hand-picked one.
  * @author Timur Saglam
  */
-public class Language extends de.jplag.emf.Language { // currently not included in the CLI
+public class DynamicMetamodelLanguage extends MetamodelLanguage { // currently not included in the CLI
```

Comment:
Use DynamicEmfLanguage for the dynamic module, and EmfLanguage for the other one.